docs/guides/graphql/menu.json

```diff
@@ -0,0 +1,12 @@
+{
+  "title": "GraphQL",
```

Comment:
Same here. Probably just call it "API (GraphQL)"

Reply:
Agreed